trajectory_tracker/src/trajectory_tracker.cpp

```diff
@@ -119,30 +117,27 @@ class TrackerNode
   double goal_tolerance_lin_vel_;
   double goal_tolerance_ang_vel_;
 
-  ros::Subscriber sub_path_;
-  ros::Subscriber sub_path_velocity_;
-  ros::Subscriber sub_vel_;
-  ros::Subscriber sub_odom_;
-  ros::Publisher pub_vel_;
-  ros::Publisher pub_status_;
-  ros::Publisher pub_tracking_;
-  ros::NodeHandle nh_;
-  ros::NodeHandle pnh_;
-  tf2_ros::Buffer tfbuf_;
-  tf2_ros::TransformListener tfl_;
-  ros::Timer odom_timeout_timer_;
+  rclcpp::Subscription<nav_msgs::msg::Path>::SharedPtr sub_path_;
+  rclcpp::Subscription<trajectory_tracker_msgs::msg::PathWithVelocity>::SharedPtr sub_path_velocity_;
+  rclcpp::Subscription<std_msgs::msg::Float32>::SharedPtr sub_vel_;
+  rclcpp::Subscription<nav_msgs::msg::Odometry>::SharedPtr sub_odom_;
+  rclcpp::Publisher<geometry_msgs::msg::Twist>::SharedPtr pub_vel_;
+  rclcpp::Publisher<trajectory_tracker_msgs::msg::TrajectoryTrackerStatus>::SharedPtr pub_status_;
+  rclcpp::Publisher<geometry_msgs::msg::PoseStamped>::SharedPtr pub_tracking_;
+  std::unique_ptr<tf2_ros::Buffer> tfbuf_;
+  std::shared_ptr<tf2_ros::TransformListener> tfl_;
+  rclcpp::TimerBase::SharedPtr odom_timeout_timer_;
+  std::shared_ptr<rclcpp::ParameterEventHandler> parameter_hander_;
+  std::vector<std::shared_ptr<rclcpp::ParameterCallbackHandle>> callback_handler_;
```

Comment:
`callback_handler_` is an array of callback handles, not a handler. 
```suggestion
  std::vector<std::shared_ptr<rclcpp::ParameterCallbackHandle>> callback_handles_;
```